resource/global/YoStarJP/resource/ocr_config.json

```diff
@@ -22,7 +22,7 @@
         ["へ", "ベ", "ペ", "ヘ", "べ", "ぺ"],
         ["ホ", "ボ", "ポ"],
         ["り", "リ"],
-        ["o", "O"],
+        ["o", "O", "о"],
```

Comment:
This is crazy OCR type of precision. 
When it's not needed the OCRer will find everything it doesn't need to